src/components/App.tsx

```diff
@@ -25,14 +36,18 @@ const App: React.FC<AppProps> = (props) => {
     return <ErrorContainer />
   }
 
-  if (props.rendererVisible) {
-    return <React.Fragment />
+  if (props.sessionReady) {
```

Comment:
This _also_ needs to disappear once the renderer is visible.
```suggestion
  // We had a performance regression in the past where the loading GIF behind the renderer
  // raised CPU usage levels and reduced the FPS significantly
  if (props.rendererVisible) {
    return <React.Fragment />
  }
  if (props.sessionReady) {
```

Reply:
It does, but that is control in other place so it can disappear with an animation